manifests/init.pp

```diff
@@ -248,7 +250,7 @@
   # Determine hiera version
   case $hiera_version {
     '5':  { if ($hierarchy !~ Hiera::Hiera5_hierarchy) {
-              fail('${hierarchy} should be an array of hash')
+              fail("${hierarchy} should be an array of hash")
```

Comment:
I don't think this is desirable, perhaps instead of `${hierarchy}` it should read `hiera::hierarchy` or `The "hierarchy" parameter` ?

Reply:
I think this is due to an updated puppet lint autofix, it's not really changed code relevant to this PR.

Reply:
Right, my suggested fixes should satisfy puppet-lint, but not change the desired output to include the contents of the variable.